package.json

```diff
@@ -9,6 +9,8 @@
   ],
   "scripts": {
     "build": "node ./scripts/build.js",
+    "features-update": "node ./scripts/update-scratch-def-features.js",
+    "settings-update": "node ./scripts/update-scratch-def-settings.js",
```

Comment:
interesting...should we automate this via a pre-commit hook or something?

Reply:
yeah, I was thinking we should automate something! there's a few repos that count on major updates... if we could centralize some sort of event to get sent out when that happens there's a few different scripts we could run across repos to keep up to date.